### DISCUSSION TITLE
Email limits

### DISCUSSION THREAD

**Discussion by rodrigo14032004** [supabase/supabase#28123] [Questions]
When I activate the option to confirm the email, I am almost unable to create any accounts because it reaches the limit, I am going to launch an application that will have many registrations, how can I prevent this from happening?
![2](https://github.com/user-attachments/assets/b091402d-8152-41ef-ac1c-7ba743f2c247)

**Comment by GaryAustin1** [comment]
The built in SMTP is only 3 per hour.  You need your own SMTP provider.
https://supabase.com/docs/guides/auth/auth-smtp
